terraform/custom_domains/environment_domains/config/ccbl_preproduction.tfvars.json

```diff
@@ -0,0 +1,15 @@
+{
+  "enable_alerting": false,
+  "hosted_zone": {
+    "check-the-childrens-barred-list.education.gov.uk": {
+      "front_door_name": "s189p01-ccbl-edu-domains-fd",
+      "resource_group_name": "s189p01-ccbldomains-rg",
+      "domains": ["preproduction"],
+      "cached_paths": ["/assets/*"],
+      "environment_short": "preproduction",
```

Comment:
I can't remember where this `environment_short` is used now, but I would say this should be `"pp"` to match where you've got `CONFIG_SHORT=pp` in `global_config/preproduction.sh`